CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 3.0)
 
+# Include guard for including this project multiple times
+if(TARGET Headers)
+  return()
+endif()
+
 project(OpenCLHeaders
   VERSION 2.2
```

Comment:
Is this version correct? Might not be related to this PR, or is it? I am no CMake expert, but has the behavior of the CMake package changed significantly enough to warrant a package version change? Or should the version match those of the headers?

Reply:
This specific PR hasn't changed much, so I don't think so. I think bumping this version should be the subject of a separate discussion. IMHO it should be 3.0.x, but it should be discussed with the group.

Reply:
Thanks for the insight.